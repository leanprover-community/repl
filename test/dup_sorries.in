{"cmd": "theorem thm1 : 1 = 1 := sorry"}

{"cmd": "theorem thm2 : 2 = 2 := sorry", "env": 0}
