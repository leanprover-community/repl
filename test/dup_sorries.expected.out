{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 24},
   "goal": "⊢ 1 = 1",
   "endPos": {"line": 1, "column": 29}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 12},
   "data": "declaration uses 'sorry'"}],
 "env": 0}

{"sorries":
 [{"proofState": 1,
   "pos": {"line": 1, "column": 24},
   "goal": "⊢ 2 = 2",
   "endPos": {"line": 1, "column": 29}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 12},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

